pwndbg/commands/heap.py

```diff
@@ -495,51 +495,51 @@ def find_fake_fast(addr, size):
 
 
 vis_heap_chunks_parser = argparse.ArgumentParser(description='Visualize heap chunks at the specified address')
-vis_heap_chunks_parser.add_argument('count', nargs='?', default=10,
-                    help='Number of chunks to visualize')
+vis_heap_chunks_parser.add_argument('count', type=lambda n:max(int(n, 0),1), nargs='?', default=10, help='Number of chunks to visualize')
 vis_heap_chunks_parser.add_argument('address', help='Start address', nargs='?', default=None)
-vis_heap_chunks_parser.add_argument('--naive', '-n', help='Don\'t use end-of-heap heuristics', action='store_true', default=False)
+vis_heap_chunks_parser.add_argument('--naive', '-n', help='Attempt to keep printing beyond the top chunk', action='store_true', default=False)
 
 @pwndbg.commands.ArgparsedCommand(vis_heap_chunks_parser)
 @pwndbg.commands.OnlyWhenRunning
 @pwndbg.commands.OnlyWithLibcDebugSyms
 @pwndbg.commands.OnlyWhenHeapIsInitialized
 def vis_heap_chunks(address=None, count=None, naive=None):
-    address = int(address) if address else pwndbg.heap.current.get_heap_boundaries().vaddr
     main_heap = pwndbg.heap.current
-    main_arena = main_heap.get_arena()
-
+    heap_region = main_heap.get_heap_boundaries(address)
     top_chunk = get_top_chunk_addr()
- 
-
-    unpack = pwndbg.arch.unpack
 
-    cells_map = {}
-    chunk_id = 0
     ptr_size = pwndbg.arch.ptrsize
```

Comment:
Just for quality, not really important.
```suggestion
    ptr_size = main_heap.size_sz
```